frontend/src/features/admin-form/settings/components/AuthSettingsSection/SingpassAuthOptionsRadio.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Fragment,
+  KeyboardEventHandler,
+  MouseEventHandler,
+  useCallback,
+  useState,
+} from 'react'
+import { Box, Flex, Spacer } from '@chakra-ui/react'
+
+import { FormAuthType, FormSettings, FormStatus } from '~shared/types'
+
+import Radio from '~components/Radio'
+import { Tag } from '~components/Tag'
+
+import { useMutateFormSettings } from '../../mutations'
+
+import { FORM_SINGPASS_AUTHTYPES } from './constants'
+import { EsrvcIdBox } from './EsrvcIdBox'
+
+export interface SingpassAuthOptionsRadioProps {
+  settings: FormSettings
+  isDisabled: boolean
+}
+
+const esrvcidRequired = (authType: FormAuthType) => {
```

Comment:
Looks like this is returning a boolean, would recommend following how predicates are named (is*, has*)

Reply:
resolved!